src/features/query-bar/__tests__/QueryBar-test.js

```diff
@@ -19,4 +25,20 @@ describe('features/query-bar/components/QueryBar', () => {
         });
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should render ColumnButton with columns that do not include item name', () => {
+        const wrapper = getWrapper({
+            columns: columnsWithTemplateFieldName,
+        });
+        const ColumnButton = wrapper.find('ColumnButton');
+        expect(ColumnButton.props().columns).toEqual([columnForTemplateFieldName]);
+    });
+
+    test('should render FilterButton with columns without items', () => {
```

Comment:
"columns without items" -> Should we choose naming by omission or naming by what it contains?

Reply:
We can go with "should render FilterButton with metadata columns"